.github/workflows/daily.yml

```diff
@@ -143,6 +143,31 @@ jobs:
         if: ${{ failure() }}
       - uses: ./.github/actions/build-teardown
 
+  prover-inconsistency-test:
+    runs-on: ubuntu-latest-xl
+    container:
+      image: diem/build_environment:main
+    env:
+      MESSAGE_PAYLOAD_FILE: /tmp/message
+    strategy:
+      fail-fast: false
+      matrix:
+        release-branch: [release-1.1]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.release-branch }}
+      - uses: ./.github/actions/build-setup
```

Comment:
You should copy in the cache step from all the other testing jobs,  it will ensure that you have crates locally available.

Reply:
I coped the cache step from other jobs. Does it look ok?